README.md

```diff
@@ -6,30 +6,38 @@ developing operating systems and firmware for PXE-capable platforms.
 
 It was inspired by effort required to test PC Engines apu2 platform.
 
-`./init.sh` builds container and runs it exposing port 69. There is silent
-assumption that your host doesn't have port 69 in use.
-
 Usage
 -----
 
 ```
 git clone https://github.com/3mdeb/pxe-server.git
-./init.sh
+cd pxe-server
+CLOUD_USER=<cloud-username> NFS_SRV_IP=<server-ip> ./init.sh
 ```
 
 APU2 development and testing
 ----------------------------
 
 ## Prepare iPXE
+`init.sh` script used above initializes directories, downloads PXE from Debian
+netboot, voyage netinst image and Debian 8 Jessie rootfs. It also clones the
+3mdeb netboot [repo](https://github.com/3mdeb/netboot.git)
 
-For those who want to test apu2 I advise to use [3mdeb/netboot](https://github.com/3mdeb/netboot) configuration.
-After finished `init.sh` as described above. Run:
+It is necessary to load nfs kernel modules before building a container:
```

Comment:
@miczyg1 why ? There should be no external dependencies in software otherwise container is pointless.

Reply:
@pietrushnic it is not gonna work if there is no support in kernel. That's how it works. There is no workaround to that.

Reply:
@miczyg1 :)